src/header.ts

```diff
@@ -179,9 +194,25 @@ export class AppHeader {
    */
   constructor(private parent: HTMLElement, options: IAppHeaderOptions = {}) {
     mixin(this.options, options);
+    this.addEUCookieDisclaimer();
     this.build();
   }
 
+  private addEUCookieDisclaimer() {
+    if(!this.options.showCookieDisclaimer) {
+      return;
+    }
+
+    const script = document.createElement('script');
```

Comment:
generell question: why did you choose to include the script on demand and not in the bundle?

Reply:
What bundle do you mean? I don't understand.

Reply:
I mean why not add a regular javascript library using npm, (lazy) import and use it. Why manually adding an external script?

Reply:
I tried that first, but the import was not straight forward and didn't work for me, because the JS implementation [loads/injects the CSS file and images itself](https://github.com/ToX82/cookie-bar/blob/master/cookiebar-latest.js#L203). Then the pathes were not correct. So as far as I understood I would have to add all necessary files manually to the webpack bundle. That's why I used the CDN version instead.

Reply:
I see, since it is an opt-in approach should be fine